### PR TITLE
Box v2 - fixed an issue where upload file ignored the folder id

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/34057

## Description
Fixed an issue where the command upload-file ignored the given folder id. Caused the file to be uploaded to the root folder by default. 

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Box/Integrations/BoxV2/BoxV2.yml [hunk 1]
```diff
@@ -3132,7 +3132,7 @@ script:
     - contextPath: Box.Folder.item_status
       description: The status of the parent of the item.
       type: String
-  dockerimage: demisto/pyjwt3:1.0.0.13142
+  dockerimage: demisto/pyjwt3:1.0.0.16907
```
Comment (amshamah419) [inline Packs/Box/Integrations/BoxV2/BoxV2.yml:3135]: We have an open issue regarding the docker image. - https://github.com/demisto/etc/issues/33818

File: Packs/Box/ReleaseNotes/2_1_1.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Box v2
+- Fixed an issue where the command ***box-upload-file*** ignored the given folder id.
```
Comment (amshamah419) [inline Packs/Box/ReleaseNotes/2_1_1.md:4]:
```suggestion
- Fixed an issue where the ***box-upload-file*** command would ignore a given folder id.
```

**Review by amshamah419 — APPROVED** [review]
Looks good, but will require a force merge due to the docker image - @yuvalbenshalom

File: Packs/Box/ReleaseNotes/2_1_1.md (hunk 2 above)
Comment (abaumgarten) [reply Packs/Box/ReleaseNotes/2_1_1.md]: Done

File: Packs/Box/Integrations/BoxV2/BoxV2.yml (hunk 1 above)
Comment (abaumgarten) [reply Packs/Box/Integrations/BoxV2/BoxV2.yml:3135]: Done

**Comment by abaumgarten** [comment]
@yuvalbenshalom In order to merge we will need a `force merge`, an UT was added.

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**